rest-service/manager_rest/rest/resources_v3_1/cluster_status.py

```diff
@@ -68,15 +72,37 @@ def put(self, node_id, model, node_type):
                 'The given node id {} is invalid'.format(node_id))
         report_time = parse_datetime_string(report_dict['timestamp'])
         self._verify_timestamp(report_time)
-        path = '{status_path}/{node_type}_{node_id}.json'.format(
-            status_path=STATUSES_PATH, node_type=node_type, node_id=node_id)
-        if os.path.exists(STATUSES_PATH):
-            self._verify_report_newer_than_current(report_time, path)
+        status_path = '{path}/{node_type}_{node_id}.json'.format(
+            path=CLUSTER_STATUS_PATH, node_type=node_type, node_id=node_id)
+        if os.path.exists(CLUSTER_STATUS_PATH):
+            self._verify_report_newer_than_current(report_time, status_path)
         else:
-            os.makedirs(STATUSES_PATH)
-        with open(path, 'w') as report_file:
+            os.makedirs(CLUSTER_STATUS_PATH)
+        with open(status_path, 'w') as report_file:
```

Comment:
the writing should also probably be implemented in the `manager` module, where the reading is, instead of here